<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PuzzleGamesClass</class>
 <widget class="QMainWindow" name="PuzzleGamesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Puzzle Games</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60,60,60);
</string>
   </property>
   <widget class="QStackedWidget" name="MainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="GameSelectWindow">
     <widget class="QLabel" name="SelectGameLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Select game</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="bar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>48</y>
        <width>601</width>
        <height>4</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(140,140,140);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="GamesFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>581</width>
        <height>651</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60,60,60);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="GamesLayoutTop">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="MSweeperSelectFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(100,100,100);
border-radius: 10px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="MSweeperSelectLabel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Minesweeper</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="MSweeperPlayBtn">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>150</y>
              <width>131</width>
              <height>41</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
color: rgb(255, 255, 255);

padding: 0px 0px 0px 0px;
background-color: rgb(80,80,80);</string>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QFrame" name="MSweeperPicFrame">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>111</width>
              <height>101</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80,80,80);
border-radius: 0px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="MSweeperPic">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>91</width>
               <height>81</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="PuzzleGames.qrc">:/minesweeperPicture/minesweeperPicture.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="BShipSelectFrame">
           <property name="minimumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(100,100,100);
border-radius: 10px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="BShipSelectLabel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Battleship</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="BShipPlayBtn">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>150</y>
              <width>131</width>
              <height>41</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
color: rgb(255, 255, 255);

padding: 0px 0px 0px 0px;
background-color: rgb(80,80,80);</string>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QFrame" name="BShipPicFrame">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>111</width>
              <height>101</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80,80,80);
border-radius: 0px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="BShipPic">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>91</width>
               <height>81</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="PuzzleGames.qrc">:/battleshipPicture/battleshipPicture.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="FillSquaresSelectFrame">
           <property name="minimumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>172</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(100,100,100);
border-radius: 10px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="FillSquaresLabel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Fill the Squares</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="FillSquaresPlayBtn">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>150</y>
              <width>131</width>
              <height>41</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
color: rgb(255, 255, 255);

padding: 0px 0px 0px 0px;
background-color: rgb(80,80,80);</string>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QFrame" name="FillSquaresPicFrame">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>111</width>
              <height>101</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80,80,80);
border-radius: 0px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="FilLSquaresPic">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>91</width>
               <height>81</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>430</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="GamePlayingWindow">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60,60,60);
</string>
     </property>
     <widget class="QStackedWidget" name="GameStackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>581</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Minesweeper">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
       <widget class="QFrame" name="MSweeperTileFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>581</width>
          <height>581</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 170);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>581</width>
           <height>581</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="MSweeperMainGrid">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Battleship">
       <widget class="QFrame" name="BShipTileFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>581</width>
          <height>581</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 170);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>581</width>
           <height>581</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="BShipMainGrid">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="FillTheSquares"/>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(80,80,80);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Reset game</string>
      </property>
     </widget>
     <widget class="QLabel" name="TopRightLabel">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="TopLeftLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="StatusLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>660</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="GoBackButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>660</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(80,80,80);
border: 0px;</string>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PuzzleGames.qrc"/>
 </resources>
 <connections/>
</ui>
